keras_cv/models/stable_diffusion/stable_diffusion.py

```diff
@@ -22,11 +22,16 @@
 """
 
 import math
+from typing import List
+from typing import Optional
+from typing import Tuple
+from typing import Union
 
 import numpy as np
 import tensorflow as tf
 from tensorflow import keras
 
+import keras_cv.models.stable_diffusion.ptp_utils as ptp_utils
```

Comment:
prefer renaming to `prompt_to_prompt_utils.py`

Reply:
And have it also `import keras_cv.models.stable_diffusion.prompt_to_prompt_utils as prompt_to_prompt_utils`? @LukeWood ?